deploy/helm/sumologic/templates/events/otelcol/statefulset.yaml

```diff
@@ -46,8 +46,8 @@ spec:
 {{- end }}
 {{- if or (not (empty (include "events.statefulset.affinity" .))) .Values.otelevents.statefulset.podAntiAffinity }}
       affinity:
-{{- if not (empty (include "otelcolInstrumentation.statefulset.affinity" .)) }}
-{{ include "otelcolInstrumentation.statefulset.affinity" . | indent 8 }}
+{{- if not (empty (include "events.statefulset.affinity" .)) }}
+{{ include "events.statefulset.affinity" . | indent 8 }}
```

Comment:
might be worth adding a template test for this - https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/tests/helm/README.md#helm-template-tests

Reply:
example - https://github.com/SumoLogic/sumologic-kubernetes-collection/commit/114d97cd2a50d7cfb6db4c06456043b9ab643f0a#diff-87c4773b744518d0cc99baee3dfadffcaa566f467cbdcce2db55e26785ca3d76

Reply:
Agreed, thanks for pointing me to that!